a11y/scripts/main.json

```diff
@@ -0,0 +1,38 @@
+{
+    "defaults":
```

Comment:
This defaults section seems redundant with what is in `.pa11yci`. Do you know if we can leave this out here and just include the `urls` items?

Reply:
At the moment, there doesn't seem to be any way to inherit defaults from other files. For now, I believe we need to write 
WET (Write Everything Twice) code here.